Objects/obmalloc.c

```diff
@@ -1172,6 +1189,9 @@ static struct arena_object* unused_arena_objects = NULL;
  */
```

Comment:
Add comment documenting that they are sorted in number of free pools?

Reply:
I'd rather not, unless you feel strongly about it.  This has been documented all along, but up where the _main_ explanation of arenas lives (look at the comment block about `usable_arenas` at line 1150).

There are a bunch of moving parts here, and the style of the file is to give long-ish English explanations in uninterrupted comment blocks, with only the briefest of memory-jogging comments on declaration lines.